cli/tests/test_next_prev_commands.rs

```diff
@@ -567,6 +567,127 @@ fn test_next_editing() {
     "###);
 }
 
+#[test]
+fn test_prev_conflict() {
+    // Make the third commit our new parent.
+    let test_env = TestEnvironment::default();
+    test_env.jj_cmd_ok(test_env.env_root(), &["init", "repo", "--git"]);
+    let repo_path = test_env.env_root().join("repo");
+    let file_path = repo_path.join("content.txt");
+    test_env.jj_cmd_ok(&repo_path, &["commit", "-m", "first"]);
+    test_env.jj_cmd_ok(&repo_path, &["commit", "-m", "second"]);
+    std::fs::write(&file_path, "Other text").unwrap();
```

Comment:
nti: Since this is the first text we write to the file, I find it a bit confusing to call it "Other text". How about using "third" (since it's in the commit called "third")? Then use "second" instead of "some text" below.

Reply:
done for all.